src/main/java/io/stargate/sgv2/jsonapi/service/shredding/OrderedNamedValueContainer.java

```diff
@@ -0,0 +1,29 @@
+package io.stargate.sgv2.jsonapi.service.shredding;
+
+import java.util.Collection;
+import java.util.LinkedHashMap;
+
+/**
+ * Base implementation for a {@link NamedValueContainer} that maintains the order the named values
```

Comment:
What is the benefit/need for distinction in type hierarchy wrt ordered/unordered variants?

Looking at implementations it seems to come to `HashMap` (unordered) vs `LinkedHashMap` (insertion-ordered).
I would probably remove this distinction and simply go with ordered (`LinkedHashMap`) -- minimal performance/overhead difference, convenient for testing. Removes third of types in this package.

Reply:
tracking here to change after merge https://github.com/stargate/data-api/issues/1411